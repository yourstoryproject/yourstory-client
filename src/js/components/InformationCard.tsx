import React from 'react';
import styled from 'styled-components';

const InformationContainer = styled.div`
	background-color: rgba(255, 255, 255, 0.5);
	border-radius: 4px;
	height: auto;
	max-width: 300px;
	padding: 24px;
	margin: auto;
`;

const InformationCard: React.FunctionComponent = () => {
	return (
		<InformationContainer>
			<p>
				In this project we will explore several stories about love,
				loss, success, and failure. These stories we will explore
				intersecting identities and the many strugles we may share.
			</p>
		</InformationContainer>
	);
};

export default InformationCard;
