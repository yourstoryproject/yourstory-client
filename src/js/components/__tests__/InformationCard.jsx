import InformationCard from '../InformationCard';
import React from 'react';
import { render } from '@testing-library/react';

describe('InformationCard', () => {
	it('should render', () => {
		const component = render(<InformationCard />);
		expect(component).toMatchSnapshot();
	});
});
