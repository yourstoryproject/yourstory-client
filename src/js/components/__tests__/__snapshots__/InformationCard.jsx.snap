// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformationCard should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bdVaJa fVsBSI"
      >
        <p>
          In this project we will explore several stories about love, loss, success, and failure. These stories we will explore intersecting identities and the many strugles we may share.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bdVaJa fVsBSI"
    >
      <p>
        In this project we will explore several stories about love, loss, success, and failure. These stories we will explore intersecting identities and the many strugles we may share.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
